css/template.css

```diff
@@ -334,9 +344,8 @@ main {
   }
 }
 
-.divider {
-  margin: 24px 0;
-  height: 12px;
+.spacer {
+  margin-top: 2rem;
```

Comment:
add this to h2 class

remove bottom from the title block to fix the spacing issues